config/locales/server.en.yml

```diff
@@ -142,6 +142,18 @@ en:
   discourse_ai:
     unknown_model: "Unknown AI model"
 
+    tools:
+      custom_name: "%{name} (custom)"
+      presets:
+        browse_web_jina:
+          name: "Browse web using jina.ai"
```

Comment:
The description of the presets should probably be translated too

Reply:
tricky, but no, the description is for the LLM not for humans.

Reply:
Ah my bad I thought I saw them in the UI, but only the name is in the dropdown :+1: